gltf-json/Cargo.toml

```diff
@@ -16,3 +16,4 @@ serde_json = "1.0"
 default = []
 names = []
 extras = []
+pbr-specular-glossiness = []
```

Comment:
For familiarity I was considering using the Khronos convention for extension names, i.e. `KHR_materials_pbrSpecularGlossiness` in this case.